add support for python

DerMolly opened the pull request:
TIC-80 supports python since May 2023 or so. https://github.com/nesbox/TIC-80/pull/2184

chronoDave reviewed (commented):
Missing changes in `utils.js`, `getCommentStyle` should have a Python check as well.

DerMolly commented:
I'm unsure what you mean. I changed `getCommentStyle` in `utils.js` or do you mean I should add tests? If you mean the later could you point me to where you want those test, because I find no test that directly test `getCommentStyle` and I thought I changed the relevant test already.

chronoDave commented:
Apologies, you're right. Looks good to me
